src/components/SummaryManagement/SummaryManagement.jsx

```diff
@@ -0,0 +1,478 @@
+      fullName: `${user.firstName} ${user.lastName}`,
+      role: user.role,
+    };
+    try {
+      const response = await axios.post(
+        ENDPOINTS.SUMMARY_GROUP_TEAM_MEMBERS(selectedSummaryGroupId),
+        requestData,
+      );
+    } catch (error) {
+      console.log(error);
+    }
+    this.getTeamMembers(selectedSummaryGroupId);
+  };
+
+  onDeleteTeamMember = async deletedUserId => {
+    const requestData = {
+      _id: deletedUserId,
+    };
+
+    try {
+      const response = await axios.delete(
+        ENDPOINTS.SUMMARY_GROUP_TEAM_MEMBERS_DELETE(
+          this.state.selectedSummaryGroupId,
+          deletedUserId,
+        ),
+      );
+    } catch (error) {
+      console.log(error);
+    }
+    this.getTeamMembers(selectedSummaryGroupId);
+
+    alert(
+      'Team member successfully deleted! Ryunosuke Satoro famously said, “Individually we are one drop, together we are an ocean.” Through the action you just took, this ocean is now one drop smaller.',
+    );
+  };
+  getSummaryReceiver = async summaryGroupId => {
+    try {
+      const response = await axios.get(ENDPOINTS.SUMMARY_GROUP_SUMMARY_RECEVIER(summaryGroupId));
+      const summaryReceiver = response.data;
+      this.setState({ summaryReceiver: response.data });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  onSummaryReciverShow = async (summaryGroupId, name) => {
+    this.setState({
+      SummaryReceiverPopupOpen: true,
+      selectedSummaryGroup: name,
+      selectedSummaryGroupId: summaryGroupId,
+    });
+    this.getSummaryReceiver(summaryGroupId);
+  };
+  onSummaryReciverClose = () => {
+    this.setState({
+      selectedSummaryGroupId: undefined,
+      selectedSummaryGroup: '',
+      SummaryReceiverPopupOpen: false,
+    });
+  };
+  onAddSummaryReceiver = async (user, selectedSummaryGroupId) => {
+    const requestData = {
+      _id: user._id,
+      fullName: `${user.firstName} ${user.lastName}`,
+      role: user.role,
+      email: user.email,
+    };
+    try {
+      const response = await axios.post(
+        ENDPOINTS.SUMMARY_GROUP_SUMMARY_RECEVIER(selectedSummaryGroupId),
+        requestData,
+      );
+    } catch (error) {
+      console.log(error);
+    }
+    this.getSummaryReceiver(selectedSummaryGroupId);
+  };
+
+  onDeleteSummaryReceiver = async deletedUserId => {
+    try {
+      const response = await axios.delete(
+        ENDPOINTS.SUMMARY_GROUP_SUMMARY_RECEIVER_DELETE(
+          this.state.selectedSummaryGroupId,
+          deletedUserId,
+        ),
+      );
+    } catch (error) {
+      console.log(error);
+    }
+    this.getSummaryReceiver(this.state.selectedSummaryGroupId);
+
+    alert(
+      'Team member successfully deleted! Ryunosuke Satoro famously said, “Individually we are one drop, together we are an ocean.” Through the action you just took, this ocean is now one drop smaller.',
+    );
+  };
+  onWildCardSearch = searchText => {
+    console.log('Search text:', searchText);
```

Comment:
As wang9hu mentioned, please remove these types of console logs.

Reply:
remove as request